include/micm/util/sparse_matrix.hpp

```diff
@@ -170,12 +170,12 @@ namespace micm
 
     SparseMatrix() = default;
 
-    SparseMatrix(const SparseMatrixBuilder<T, OrderingPolicy>& builder)
+    SparseMatrix(const SparseMatrixBuilder<T, OrderingPolicy>& builder, bool empty_matrix = false)
```

Comment:
I think we should add comments to this constructor now. When it just accepted the builder object, it seemed pretty clear what it was doing. But, this flag won't be obvious to people, I don't think. (I would also change it to `indexing_only` to try to make it a little more clear)